ProteoformSuiteGUI/ExperimentExperimentComparison.cs

```diff
@@ -20,8 +20,8 @@ public ExperimentExperimentComparison()
         {
             InitializeComponent();
             this.dgv_EE_Peaks.MouseClick += new MouseEventHandler(dgv_EE_Peak_List_CellClick);
-            this.ct_EE_Histogram.MouseMove += new MouseEventHandler(ct_EE_Histogram_MouseMove);
```

Comment:
In  a way I suppose! Since you have to click the histogram line to see the x/y, it's not flickering between a bunch of super close numbers. It would be nice to hover and see what # it is but I think with as many data points as we have in ET/EE, clicking is the simpler solution